lib/chewy/rake_helper.rb

```diff
@@ -268,6 +268,24 @@ def delete_by_query_options_from_env(env)
           end
       end
 
+      def create_missing_indexes!(output: $stdout)
+        subscribed_task_stats(output) do
+          Chewy.eager_load!
+          all_indexes = Chewy::Index.descendants
+          all_indexes -= [Chewy::Stash::Journal] unless Chewy.configuration[:journal]
+          all_indexes.each do |index|
+            if index.exists?
+              output.puts "#{index.name} already exists, skipping"
```

Comment:
```suggestion
              output.puts "#{index.name} already exists, skipping" if ENV['VERBOSE']
```
Should it be less verbose by default? If we are going to run it on each deployment, that would be very noisy.

Reply:
ok, sure